serveur_calcul_python/tests/data_gen/reg_set_terr_data_store.py

```
@@ -0,0 +1,68 @@
+import geopandas as gpd
+from config import config_db as cf_db
+from shapely import Polygon
+import classes.reg_set_territory as RST
+from tests.data_gen import regulation_set_data_store as RSDS
+
+def generate_reg_set_terr():
+    tax_data_set = TDS.generate_tax_dataset()
+    top_polygon_lat = [46.5+.006,46.5+.005,46.45-.005,46.45-.005,46.5+.006]
+    polygon_lon = [-71.5-.006,-71.44+.006,-71.44+.006,-71.5-.006,-71.5-.006]
+    bottom_polygon_lat = [46.44+.005,46.44+.005,46.44-.006,46.44-.006,46.44+.005]
+    prs_1,prs_2,prs_3,prs_4 = RSDS.generate_parking_regulation_sets()
```

Ruff (0.15.12)

[warning] Unpacked variable `prs_4` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)